docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,192 @@
+and technology progress. The best approach to modifying a workflow
+is to try to recreate steps on your local system (if possible),
+identify failures and fixes, then translate fixes to the resuable
+workflow in question. If you encounter an issue that requires
+community input you can ask for help on the
+[ITK community Discourse forum](https://discourse.itk.org/).
+
+## Testing Your Changes
+
+Reusable workflows are intended to _manage continuous integration_ for
+ITK external modules in a reusable manner. At the time of writing
+there are unfortunately no good options for establishing continuous
+integration around proposed changes to reusable workflows themselves,
+therefore it is necessary to place an extra burden on the contributors and
+maintainers of `ITKRemoteModuleBuildTestPackageAction` to schedule tests themselves.
+Fortunately, this translates to severely reduced overhead required of the
+general ITK community in maintaining continuous integration on individual
+external modules.
+
+Changes in reusable workflows should be tested through application to
+ITK external modules in a user fork. The ideal repositories to use
+for testing will depend on the workflow changes under test.
+
+The [ITKSplitComponents repository](https://github.com/InsightSoftwareConsortium/ITKSplitComponents)
+is a good target for testing relatively
+simple changes. ITKSplitComponents is a small header-only external module
+with a minimum testing suite, Python wrappings, and an example Jupyter Notebook.
+Do the following to test against ITKSplitComponents or another ITK external module:
+1. Commit your changes to `ITKRemoteModuleBuildTestPackageAction` and
+   push to make them available on your GitHub user account:
+```sh
+> git commit
+> git push
+```
+
+2. Fork the [ITKSplitComponents repository](https://github.com/InsightSoftwareConsortium/ITKSplitComponents)
+    to your GitHub user account and clone to your local machine:
+```sh
+> cd ..
+> git clone git@github.com:<your-username>/ITKSplitComponents.git
+> cd ITKSplitComponents
+```
+
+3. Create a development branch for your proposed changes:
+```sh
+> git checkout -b "my-cool-feature"
+```
+
+4. Update the external module workflow `.yml` file to reference your changes:
+```yml
+name: Build, test, package
+
+on: [push,pull_request]
+
+jobs:
+  cxx-build-workflow:
+    uses: <your-username>/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-cxx.yml@<your-dev-branch>
+    with:
+      itk-cmake-options: '-DITK_BUILD_DEFAULT_MODULES:BOOL=OFF -DITKGroup_Core:BOOL=ON'
+
+  python-build-workflow:
+    uses: <your-username>/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-package-python.yml@<your-dev-branch>
+    secrets:
+      pypi_password: ${{ secrets.pypi_password }}
+```
+
+5. Commit and push your changes:
+```sh
+> git commit
+> git push
+```
+
+6. Visit `https://github.com/<your-username>/ITKSplitComponents/actions` to view running GHA jobs.
```

Comment:
First time the GHA acronym is used. Should be defined early (and maybe reused to save typing),

Reply:
Given that the term "GitHub Actions" is not excessively used in this document I have opted to remove the abbreviation and use the full term throughout for clarity.